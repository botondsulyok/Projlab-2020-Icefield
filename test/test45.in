field spawn 0
entity esk Eskimo spawn
item food1 Food spawn
stats spawn
stats esk
storm esk
storm esk
stats esk
dig esk
stats spawn
stats esk