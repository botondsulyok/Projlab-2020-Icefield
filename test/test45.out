id: spawn
type: iceunit
building: plains0
snow: 0
pickable: food1
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 3
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 4
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

